lib/store/package.json

```diff
@@ -56,6 +56,10 @@
     "ts-dedent": "^2.0.0",
     "util-deprecate": "^1.0.2"
   },
+  "peerDependencies": {
+    "react": "^16.8.0 || ^17.0.0",
+    "react-dom": "^16.8.0 || ^17.0.0"
+  },
```

Comment:
In principle this is fine, I fear it might introduce global React types for Vue projects.

Reply:
no should be fine, we have these in other packages that are not react-specific already.